src/runner.ts

```diff
@@ -282,11 +305,26 @@ export class GitHubRunners extends Construct {
       .when(stepfunctions.Condition.isNotPresent('$.labels.self-hosted'), new stepfunctions.Succeed(this, 'No'))
       .otherwise(work);
 
+    const logGroup = new logs.LogGroup(
+      this,
+      'Logs',
+      {
+        logGroupName: props?.logGroupName ?? undefined,
+        retention: props?.logRetention ?? logs.RetentionDays.ONE_MONTH,
+        removalPolicy: props?.logRemovalPolicy ?? RemovalPolicy.DESTROY,
+      },
+    );
+
     return new stepfunctions.StateMachine(
       this,
       'Runner Orchestrator',
       {
         definition: check,
+        logs: {
+          destination: logGroup,
+          level: stepfunctions.LogLevel.ALL,
```

Comment:
Should we add an option to reduce the verbosity? What's even printed for a level like `DEBUG`?

Reply:
makes sense but at some point we can again talk about if it makes sense to add a logOptions property optional as I had it at the beginning

Reply:
@kichik ok for you to add optional logOptions as property ?

Reply:
Yeah we can add optional log options parameter. It would probably be the nicest looking option.

But I would rather not use that specific `LogOptions` interface as it's different than existing code in the sense that you have to pass in an existing log group.

Reply:
@kichik  
ok added an additional interface for the LogOptions especially for the runner.
please have a look

Reply:
I haven`t added any unit tests for the moment nor have I deployed it so far.
Let me first know if this is what you expect ...

Reply:
Yeah exactly what I meant. Thanks.